update_pred_1 threw exception.
Final value of Globals = 3
